app/components/provider_interface/interview_and_course_summary_component.html.erb

```diff
@@ -5,10 +5,16 @@
         <div class="app-summary-card__actions">
           <ul class="app-summary-card__actions-list">
             <li class="app-summary-card__actions-list-item">
-              <%= govuk_link_to 'Change details', edit_provider_interface_application_choice_interview_path(application_choice, interview), class: 'app-summary-card__actions-list-item govuk-link--no-visited-state' %>
+              <%= link_to edit_provider_interface_application_choice_interview_path(application_choice, interview), class: 'app-summary-card__actions-list-item govuk-link govuk-link--no-visited-state' do %>
```

Comment:
I missed this in my previous review. Is there any reason we are no longer using `govuk_link_to` here?